client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts

```diff
@@ -226,4 +227,26 @@ export class AgendaListComponent extends ListViewBaseComponent<ViewItem> impleme
     public csvExportItemList(): void {
         this.csvExport.exportItemList(this.dataSource.data);
     }
+
+    // dirty, not observing changes
+    public getHierarchyLevel(item: ViewItem): number {
+        if (!item.item.parent_id) {
+            return 0;
+        }
+        return 1 + this.getHierarchyLevel(this.repo.getViewModel(item.item.parent_id));
+    }
+
+    // TODO Docu
+    public getHierarchyClass(item: ViewItem): string {
```

Comment:
Please do not use classes here, because this indention might be infinite. Use [ngStyle] and set margin and font per item.

Reply:
Most browsers won't support infinite indentation. I'd rather have 'for or five levels defined, the rest in a 'further class'. But as said, it's still a dummy, I'll wait for design feedback

Reply:
In the end, I did as you wanted